test/parallel/test-dotenv-edge-cases.js

```diff
@@ -2,10 +2,12 @@
 
 const common = require('../common');
 const assert = require('node:assert');
+const path = require('node:path');
```

Comment:
```suggestion
const fixtures = require('../common/fixtures');
const path = require('node:path');
```

Reply:
Didn't know `common/fixtures` was a thing! Thank you.